source/base/ReferenceCounter.ooc

```diff
@@ -1,20 +1,29 @@
 import threading/Thread
 import Synchronized
 
-ReferenceCounter: class extends Synchronized {
+ReferenceCounter: class {
 	_target: Object
 	count: static Int = 0
 	_count: Int = 0
 	_kill := false
+	_lock := Mutex new(MutexType Safe)
```

Comment:
So... if you want an unsafe one, you need to first create the `ReferenceCounter`, then call `isSafe(false)`?